### ISSUE TITLE
Can't compile for Debian 8 or Debian 10

### ISSUE THREAD

**Issue by LupusiAlba** [otland/forgottenserver#3158]
### Before creating an issue, please ensure:
- [ ] This is a bug in the software that resides in this repository, and not a
      support matter (use https://otland.net/forums/support.16/ for support)
- [ ] This issue is reproducible without changes to the C++ code in this repository

### Steps to reproduce (include any configuration/script required to reproduce)
1. cd forgottenserver mkdir
2. build && cd build
3. cmake ..

### Expected behaviour
Compiled successfully. 

### Actual behaviour
-- The C compiler identification is GNU 4.9.2
-- The CXX compiler identification is GNU 4.9.2
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found Crypto++: /usr/include  
-- MySQL Include dir: /usr/include/mysql  library dir: /usr/lib/x86_64-linux-gnu
-- MySQL client libraries: /usr/lib/x86_64-linux-gnu/libmysqlclient.so;-ldl
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE  
-- Found PugiXML: /usr/include  
-- Found LuaJIT: /usr/lib/x86_64-linux-gnu/libluajit-5.1.so;/usr/lib/x86_64-linux-gnu/libm.so (found version "2.0.3") 
CMake Error at /root/cmake-3.18.1-Linux-x86_64/share/cmake-3.18/Modules/FindPackageHandleStandardArgs.cmake:165 (message):
  Could NOT find Boost (missing: date_time) (found suitable version "1.55.0",
  minimum required is "1.53.0")
Call Stack (most recent call first):
  /root/cmake-3.18.1-Linux-x86_64/share/cmake-3.18/Modules/FindPackageHandleStandardArgs.cmake:458 (_FPHSA_FAILURE_MESSAGE)
  /root/cmake-3.18.1-Linux-x86_64/share/cmake-3.18/Modules/FindBoost.cmake:2177 (find_package_handle_standard_args)
  CMakeLists.txt:63 (find_package)

-- Configuring incomplete, errors occurred!
See also "/root/forgottenserver/build/CMakeFiles/CMakeOutput.log".
See also "/root/forgottenserver/build/CMakeFiles/CMakeError.log".

### Environment
Debian 8
Debian 10

**Comment by EPuncker** [comment]
make sure you followed ALL the steps on the guide https://github.com/otland/forgottenserver/wiki/Compiling-on-Debian-GNU-Linux

**Comment by LupusiAlba** [comment]
Whole procedure was replicated twice on each build. Furthermore, colleague had exactly the same error log on different device upon trying to follow given instructions.

Ubuntu 20.4 compiles correctly.

**Comment by ranisalt** [comment]
You are missing libboost-date-time-dev I believe

**Comment by ranisalt** [comment]
> You are missing libboost-date-time-dev I believe

@LupusiAlba can you confirm that works or not?

**Comment by LupusiAlba** [comment]
Hello there, sorry for not providing reply. I did download all packages which were provided in the "compiling on Debian" section, it did not work however, so I ditched debian for ubuntu. Here it worked as it was intended, however the cmake version package given there is too old (3.5 vs 3.10 being necessary to complete compiling), so I had to download newer version. Besides from it, it's OK. Now I have different issue, for which I made separate report.
If you want to further investigate the issue with Debian, that's up to you, I suppose.

Thanks
